model_read_only/models/models.py

```diff
@@ -1,14 +1,29 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo import _, api, exceptions, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class ModelReadonlyRestriction(models.Model):
+    _name = "model.readonly.restriction"
+
+    model_id = fields.Many2one("ir.model")
+    model_name = fields.Char(related="model_id.model")
+    restriction_domain = fields.Char("Domain")
+    group_ids = fields.Many2many(
+        "res.groups",
+        relation="res_group_model_readonly_restriction_rel",
+        string="Groups",
+    )
```

Comment:
**suggestion**: If I understand correctly:

- if `restriction_domain` is empty, the restriction is always active
- if `group_ids` is empty, the restriction applies to everyone

could you add this bit of documentation to the `help` of the fields as well as the README?